example/rcnn/rcnn/core/tester.py

```diff
@@ -15,13 +15,14 @@
 # specific language governing permissions and limitations
 # under the License.
 
-import cPickle
+from six.moves import cPickle
```

Comment:
can you do it without adding six?

Reply:
this example already requires few libraries to be installed. Installing six doesn't seem too hard? Besides not using six means using ugly if else statements for different versions of python